driver.py

```diff
@@ -72,12 +81,18 @@ def get_table(table_name):
     return result
 
 
-def read_metadata(filename):
```

Comment:
Why was "filename" changed to "f"? I feel it's a little clearer to leave it as "filename" from a first-pass-understanding-the-code perspective.